ccd-user-profile-importer/templates/user-profile-importer.yaml

```diff
@@ -6,44 +7,33 @@ metadata:
     {{- ( include "ccdupi.labels" . ) | indent 4 }}
   annotations:
     helm.sh/hook: post-install,post-upgrade
-    helm.sh/hook-weight: "5"
-    helm.sh/hook-delete-policy: before-hook-creation  # Move to hook-succeeded after first charts are done 
+    helm.sh/hook-weight: "-5"
```

Comment:
Why is the weight changed from old ccd chart?

Reply:
changed back to original weight